website/blog/2024-04-04-angular-upgrade-top-5-tips.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Angular Upgrade Journey: Top 5 things to keep in mind"
+author: Sairama Krishna Bonala
+---
+
+Taking up an Angular upgrade can be both exciting and challenging. As technology evolves, it's essential to keep up to date for better performance and security. This guide explains key considerations and best practices to make the Angular upgrade journey smoother and successful.
+
+<!--truncate-->
+
+## 1. Check Compatibility
+- Check [Support timelines](https://angular.io/guide/releases#support-window) for the Active and LTS versions.
+- Consider upgrading Angular in incremental versions especially if the project is moving from a significantly older version. 
+- Angular provides comprehensive documentation on breaking changes between versions. Utilize tools like the [Angular Update Guide](https://update.angular.io/), which guides the necessary changes. Look specifically for any breaking changes or deprecated features.
+- Consider creating a separate branch for the upgrade process to avoid disrupting the main development workflow until the upgrade is complete.
```

Comment:
Aren't we automatically upgrading to Angular 16? Are we expecting users to create branches and so on?